public/app/features/alerting/unified/utils/rule-form.ts

```diff
@@ -119,6 +127,18 @@ export const getDefautManualRouting = () => {
   return manualRouting !== 'false';
 };
 
+function getDefaultEditorSettings() {
+  const editorSettingsEnabled = config.featureToggles.alertingQueryAndExpressionsStepMode ?? false;
+  if (!editorSettingsEnabled) {
+    return undefined;
+  }
+  //then, check in local storage if the user has saved last rule with simplified query editor
+  const queryEditorSettings = localStorage.getItem(SIMPLIFIED_QUERY_EDITOR_KEY);
+  return {
+    simplifiedQueryEditor: queryEditorSettings !== 'false',
```

Comment:
(Maybe explicitly check `=== 'true'` rather than `not 'false'`?)

Reply:
I'm using this expressions as I want to return true if it's undefined.